xbmc/guilib/GraphicContext.cpp

```diff
@@ -369,19 +369,19 @@ bool CGraphicContext::IsValidResolution(RESOLUTION res)
 }
 
 // call SetVideoResolutionInternal and ensure its done from mainthread
-void CGraphicContext::SetVideoResolution(RESOLUTION res, bool forceUpdate)
+void CGraphicContext::SetVideoResolution(RESOLUTION res, bool forceUpdate, bool callWindowing)
```

Comment:
It's the difference between when Kodi asks for resolution change (because someone other than windowing calls `SetVideoResolution`) and when compositor asks for size change https://github.com/pkerling/xbmc/pull/44/files#diff-3aafa5e2a705eda00659cf974e3a19b6R838